confgenerator/confgenerator.go

```diff
@@ -225,8 +225,27 @@ func defaultStackdriverOutputs() (stackdrivers []*conf.Stackdriver) {
 	}
 }
 
-func generateFluentBitConfigs(logging *logging, logsDir string, stateDir string) (string, string, error) {
-	fbTails := defaultTails(logsDir, stateDir)
+func getUserAgent(prefix string, hostInfo *host.InfoStat) (string, error) {
+	userAgent := map[string]string{
+	    "Prefix":     prefix,
+	    "AgentVersion": version.Version,
+	    "BuildDistro":   version.BuildDistro,
+	    "Platform": hostInfo.OS,
+	    "ShortName": hostInfo.Platform,
+	    "ShortVersion": hostInfo.PlatformVersion,
+	}
+	var userAgentTemplate = template.Must(template.New("mssql").Parse(`{{.Prefix}}/{{.AgentVersion}}-{{.BuildDistro}} (Platform={{.Platform}};ShortName={{.ShortName}};ShortVersion={{.ShortVersion}},gzip(gfe))`))
```

Comment:
Please move this template out to a separate constant outside the function (which you'll have to declare with `var` since it's an object).

Also, why is the template called "mssql"?

Reply:
done